tests/e2e/env/bin/e2e-test-integration.js

```diff
@@ -5,7 +5,7 @@ const program = require( 'commander' );
 const path = require( 'path' );
 const fs = require( 'fs' );
 const { getAppRoot } = require( '../utils' );
-const { WC_E2E_SCREENSHOTS, JEST_PUPPETEER_CONFIG } = process.env;
+const { WC_E2E_SCREENSHOTS, JEST_PUPPETEER_CONFIG, DEFAULT_TIMEOUT_OVERRIDE } = process.env;
```

Comment:
We should have this documented in the readme similar to https://github.com/woocommerce/woocommerce/blob/19c193870016670e9486ddd795b9cdcecec703f4/tests/e2e/env/README.md#test-screenshots and have a changelog entry

Reply:
Good call out. I've added those in here:

* Changelog: https://github.com/woocommerce/woocommerce/pull/30492/files#diff-713add7c2ab3532b3a59dc88e5745b85ab74ac1e94c9a144afc0a6ac656e1bf0R10
* README: https://github.com/woocommerce/woocommerce/pull/30492/files#diff-fb9ea0584cd5ee98bcca789f51520e4c7950d7d6a1785d81d2d81e39d71a94aeR90